packages/web/src/scss/components/Grid/index.html

```diff
@@ -340,6 +340,61 @@ <h2 class="docs-Heading">Responsive Centered Grid Item</h2>
       </div>
 
     </div>
-  </section>
+</section>
+
+<section class="UNSTABLE_Section">
+  <h2 class="docs-Heading">Align Items</h2>
```

Comment:
I think this should be `Alignment` so it better matches other headings of this kind.